src/System.Runtime.Serialization.Formatters/ref/System.Runtime.Serialization.Formatters.cs

```diff
@@ -193,23 +193,6 @@ public sealed partial class BinaryFormatter : System.Runtime.Serialization.IForm
         public System.Runtime.Serialization.ISurrogateSelector SurrogateSelector { get { throw null; } set { } }
         public System.Runtime.Serialization.Formatters.FormatterTypeStyle TypeFormat { get { throw null; } set { } }
         public object Deserialize(System.IO.Stream serializationStream) { throw null; }
-        public object Deserialize(System.IO.Stream serializationStream, System.Runtime.Remoting.Messaging.HeaderHandler handler) { throw null; }
         public void Serialize(System.IO.Stream serializationStream, object graph) { }
-        public void Serialize(System.IO.Stream serializationStream, object graph, System.Runtime.Remoting.Messaging.Header[] headers) { }
-        public object UnsafeDeserialize(System.IO.Stream serializationStream, System.Runtime.Remoting.Messaging.HeaderHandler handler) { throw null; }
     }
-} // end of System.Runtime.Serialization.Formatters.Binary
-namespace System.Runtime.Remoting.Messaging
-{
-    public partial class Header
```

Comment:
see https://github.com/dotnet/standard/commit/16d0fba6f07e72f3fa189aec296b3e3bebca5449 where it is removed from netstandard. i am not sure what the extensions actually mean